Source/Core/DolphinQt2/Debugger/CodeWidget.cpp

```diff
@@ -238,9 +238,9 @@ void CodeWidget::OnSelectFunctionCallers()
   Update();
 }
 
-void CodeWidget::SetAddress(u32 address)
+void CodeWidget::SetAddress(u32 address, bool update)
```

Comment:
enum class please, otherwise it's not immediately obvious what the false/true means when you call SetAddress.